gx_suite_definitions/team_info.ipynb

```diff
@@ -0,0 +1,242 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.read_json(team_info_data_file)\n",
+    "nested_columns = ['members']\n",
+    "df = GreatExpectationsRunner.convert_nested_columns_to_json(df, nested_columns)\n",
+    "validator = context.sources.pandas_default.read_dataframe(df)\n",
+    "validator.expectation_suite_name = \"team_info\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Add Expectations to Validator Object For Each Column"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# team\n",
+    "validator.expect_column_values_to_be_of_type(\"team\", \"str\")\n",
+    "validator.expect_column_values_to_not_be_null(\"team\")\n",
+    "validator.expect_column_value_lengths_to_be_between(\"team\", min_value=3, max_value=50)\n",
+    "validator.expect_column_values_to_be_in_set(\n",
+    "    \"team\", {\n",
+    "        \"Columbia-Rush\",\n",
+    "        \"Duke\",\n",
+    "        \"Emory\",\n",
+    "        \"Harvard-MIT\",\n",
+    "        \"MSSM - Roussos Lab\",\n",
+    "        \"MSSM - Zhang Lab\",\n",
+    "        \"Sage Bionetworks\",\n",
+    "        \"Mayo-UFL-ISB\",\n",
+    "        \"Chang Lab\", \n",
+    "        \"JAX-VUMC-UW Resilience\",\n",
+    "        \"ASU\",\n",
+    "        \"Longo Lab\",\n",
+    "        \"Duke BARU\", \n",
+    "        \"Emory-Sage-SGC\",\n",
+    "        \"IUSM-Purdue\",\n",
+    "        \"Mayo\"\n",
+    "    }\n",
+    ")\n",
+    "validator.expect_column_values_to_be_unique(\"team\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# team_full\n",
+    "validator.expect_column_values_to_be_of_type(\"team_full\", \"str\")\n",
+    "validator.expect_column_values_to_not_be_null(\"team_full\")\n",
+    "validator.expect_column_value_lengths_to_be_between(\"team_full\", min_value=10, max_value=100)\n",
+    "validator.expect_column_values_to_match_regex(\"team_full\", \"^[a-zA-Z0-9-() ]+$\")\n",
+    "validator.expect_column_values_to_be_unique(\"team_full\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# program\n",
+    "validator.expect_column_values_to_be_of_type(\"program\", \"str\")\n",
+    "validator.expect_column_values_to_be_in_set(\"program\", {'Community Contributed', None, 'AMP-AD', 'Resilience-AD', 'TREAT-AD'})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# description\n",
+    "validator.expect_column_values_to_be_of_type(\"description\", \"str\")\n",
+    "validator.expect_column_values_to_not_be_null(\"description\")\n",
+    "validator.expect_column_values_to_match_regex(\"description\", \"^[a-zA-Z0-9-(),.':/ ]+$\")\n",
```

Comment:
Nit: Are there any characters that are NOT allowed in this field? i.e. do we need a regex at all?

Reply:
So I think this guards against non UTF-8 characters, but if it isn;t necessary I am happy to remove it

Reply:
@JessterB thoughts?

Reply:
We have historically had problems with UTF-8 characters (mainly Microsoft "smart" quotes, and alpha/beta/etc. characters) in user-submitted content like team descriptions, nominations, etc. These get transformed into "� (REPLACEMENT CHARACTER, [U+FFFD]" during processing. Ideally we will eventually detect the unsupported characters during source file validation (unless anyone can think of a way to improve ADT to handle them?), and ADT should never see them. We aren't there yet though, so catching these before we display them in Agora would be a good thing. One option would be to change the regex to just specifically disallow U+FFFD. Or we could wait to see if the current regex is problematic before trying to adjust it.

Reply:
I think what I have here _should_ prevent the issues you are describing. So maybe we just see how it goes?

Reply:
Sounds reasonable to me.